src/helpers/task-metadata.ts

```diff
@@ -108,3 +108,45 @@ function parseTimeLabel(
 
   return taskTimeEstimate;
 }
+
+export function parsePriorityLabel(
+  labels: (
```

Comment:
Pretty sure this type is defined within `octokit`.

Reply:
This type wasn't defined exactly as is, I had to change one of the key value properties and make every key optional